go/gildedrose/updatestrategy/conjured.go

```diff
@@ -0,0 +1,20 @@
+package updatestrategy
+
+type conjuredUpdateStrategy struct {
+	defaultSellinUpdateStrategy // doing this assigns a default sellin update strategy
+}
+
+func (a *conjuredUpdateStrategy) GetUpdatedQuality(quality, sellIn int) (updatedQuality int) {
```

Comment:
I am assuming -2 instead of -1 and -4 instead of -2 is what meant by twice as fast compared to normal items.
hence wrote this logic